src/state/selectors.js

```diff
@@ -1,22 +1,25 @@
 // Use createSelector for any reducer which returns a computed object
 import { createSelector } from 'reselect';
-
 import objectToList from '../utils/object-to-list';
+import listToObject from '../utils/list-to-object';
+import log from '../utils/log';
 
 export const getResponses = state => state.responses.dictionary;
 export const getSelected = state => state.selected;
 export const getAggregations = state => state.summary.aggregations;
 export const getQuestions = state => state.questions.dictionary;
+export const getFields = state => state.fields.data;
 
 export const getIsFetching = state => [
   'questions',
   'responses',
-  'summary'
+  'summary',
+  'fields'
 ].reduce((isFetching, storeKey) => isFetching || state[storeKey].isFetching, false);
 
 export const getQuestionsList = createSelector(getQuestions, objectToList);
-
 export const getResponsesList = createSelector(getResponses, objectToList);
+export const getFieldsData = createSelector(getFields, listToObject('field-id (don\'t change!)'));
```

Comment:
The results I get from google sheets is an array, I convert it to an object. I was just following @kadamwhite's example above. What would be helpful here?